package.json

```diff
@@ -39,13 +39,13 @@
     "nw-gyp": "^3.6.6",
     "rc": "^1.2.8",
     "run-waterfall": "^1.1.7",
-    "tar-stream": "^3.1.6"
+    "tar-stream": "^3.1.7"
   },
   "devDependencies": {
     "a-native-module": "^1.0.0",
-    "rimraf": "^4.4.1",
+    "rimraf": "^5.0.5",
```

Comment:
Can we remove this dependency entirely by switching to `fs.rmSync(path, { recursive: true })`?

Reply:
It doesn't work well. See #319